Added tip section for group components (#6348)
yamada-ui/yamada-ui: www/contents/components/(components)/group.mdx

<Group />
```

:::tip

> If you don't need to group child elements, consider using [HStack](/docs/components/h-stack) or [VStack](/docs/components/v-stack).

Reviewer: This wording is a little confusing because `HStack` and `VStack` also group children in a layout sense. The real distinction seems to be whether the children need to be attached/treated as one grouped control. I think this should frame `HStack`/`VStack` as the simpler spacing/layout option instead.
```suggestion
If you only need to arrange child elements with spacing, consider using [HStack](/docs/components/h-stack) or [VStack](/docs/components/v-stack).
```
